File: dsa_algo/leetcode/arrays/lc/java/removeDuplicates.java
public class removeDuplicates {
    public static int remove_duplicates(int[] nums) {
       int insert_loc = 1;
       for(int i = 1; i < nums.length; i++) 
       {
            if(nums[i] != nums[i - 1])
            {
                nums[insert_loc] = nums[i];
                insert_loc++;
            }
       }
       return insert_loc;
    }
}

File: dsa_algo/leetcode/arrays/lc/python/remove_duplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        insert_loc = 1
        length = len(nums)
        for i in range(1, length):
            if nums[i] != nums[i - 1]:
                nums[insert_loc] = nums[i]
                insert_loc += 1
        return insert_loc